awx/api/views/__init__.py

```diff
@@ -708,6 +708,12 @@ class ExecutionEnvironmentJobTemplateList(SubListAPIView):
     relationship = 'unifiedjobtemplates'
 
 
+class ExecutionEnvironmentCopy(CopyAPIView):
+    
```

Comment:
Are we updating `summary_fields` to add the copy field as part of the `user_capabilities`?

Reply:
Good call, we should do that.

Reply:
done!